start-tests.sh

```diff
@@ -7,97 +7,209 @@
 ####################
 ## COLORS
 ####################
-PURPLE="\033[0;35m"
 CYAN="\033[0;36m"
+GREEN="\033[0;32m"
+PURPLE="\033[0;35m"
+RED="\033[0;31m"
 YELLOW="\033[0;33m"
 NC="\033[0m"
 
 log_color () {
   case $1 in
-    yellow)
-      echo -e "${YELLOW}$2 ${NC}"$3
-    ;;
     cyan)
       echo -e "${CYAN}$2 ${NC}"$3
     ;;
+    green)
+      echo -e "${GREEN}$2 ${NC}"$3
+    ;;
     purple)
       echo -e "${PURPLE}$2 ${NC}"$3
     ;;
+    red)
+      echo -e "${RED}$2 ${NC}"$3
+    ;;
+    yellow)
+      echo -e "${YELLOW}$2 ${NC}"$3
+    ;;
   esac
 }
 
 log_color "cyan" "Initiating Search E2E tests...\n"
 
 if [ -z "$BROWSER" ]; then
-  log_color "purple" "BROWSER" "not exported; setting to 'chrome' (options available: 'chrome', 'firefox')"
+  log_color "purple" "BROWSER" "not exported; setting to 'chrome' (options available: 'chrome', 'firefox')\n"
   export BROWSER="chrome"
+else
+  log_color "purple" "BROWSER set (running in $BROWSER)\n"
 fi
 
 # Load test config mounted at /resources/options.yaml
 OPTIONS_FILE=/resources/options.yaml
-USER_OPTIONS_FILE=./options.yaml
+USER_OPTIONS_FILE=./resources/options.yaml
 
+# Load test kubeconfig mounted at /opt/.kube/config and /opt/.kube/import-kubeconfig
+OPTIONS_HUB_KUBECONFIG=${OPTIONS_HUB_KUBECONFIG:-'/opt/.kube/config'}
+OPTIONS_MANAGED_KUBECONFIG=${OPTIONS_MANAGED_KUBECONFIG:-'/opt/.kube/import-kubeconfig'}
+
+# Check to see if the test config options file is mounted/available.
 if [ -f $OPTIONS_FILE ]; then
   log_color "yellow" "Using test config from: $OPTIONS_FILE\n"
-  export CYPRESS_OC_IDP=`yq e '.options.identityProvider' $OPTIONS_FILE`
-  export CYPRESS_OPTIONS_HUB_BASEDOMAIN=`yq e '.options.hub.baseDomain' $OPTIONS_FILE`
-  export CYPRESS_OPTIONS_HUB_USER=`yq e '.options.hub.user' $OPTIONS_FILE`
-  export CYPRESS_OPTIONS_HUB_PASSWORD=`yq e '.options.hub.password' $OPTIONS_FILE`
   export OPTIONS_HUB_BASEDOMAIN=`yq e '.options.hub.baseDomain' $OPTIONS_FILE`
   export OPTIONS_HUB_USER=`yq e '.options.hub.user' $OPTIONS_FILE`
   export OPTIONS_HUB_PASSWORD=`yq e '.options.hub.password' $OPTIONS_FILE`
+  export OPTIONS_HUB_KUBECONTEXT=`yq e '.options.hub.kubecontext' $OPTIONS_FILE`
+  export OPTIONS_MANAGED_BASEDOMAIN=`yq e '.options.clusters[0].basedomain' $OPTIONS_FILE`
+  export OPTIONS_MANAGED_CLUSTER_NAME=`yq e '.options.clusters[0].name' $OPTIONS_FILE`
+  export OPTIONS_MANAGED_KUBECONFIG=`yq e '.options.clusters[0].kubeconfig' $OPTIONS_FILE`
```

Comment:
Adding these values to be read from the `options.yaml` file. CICD is provided the values for these variables from the file.